installer/grub-arch/install-arch

```diff
@@ -459,6 +461,21 @@ init_uefi_disk()
         create_gpt_partition $p
     done
 
+    # Update new partition node if we have nvme device
+    # Currently we only support 1 device at nvme0n1
+    nvme_devs=$(ls -d /sys/block/nvme0n1/nvme0n1p[1-7] 2> /dev/null) && {
+    for nd in $nvme_devs ; do
+        dev=$(basename $nd)
+        major=$(sed -e 's/:.*$//' $nd/dev)
+        minor=$(sed -e 's/^.*://' $nd/dev)
+        rm -f /dev/$dev
+        mknod /dev/$dev b $major $minor || {
+            echo "Problems creating /dev/$dev block device."
+            continue
+        }
+        done
+    }
```

Comment:
If the mdev hotplug helper is working properly we should not need this code here.